java/java-saltedge-example/src/com/saltedge/Program.java

```diff
@@ -1,11 +1,24 @@
 package com.saltedge;
 
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
 
 public class Program {
+    static public final String SPECTRE_PUBLIC_KEY_PATH = "saltedge_public.pem";
+    static public final String PUBLIC_KEY_ALGORITHM = "RSA";
+    static public final String CALLBACK_URL = "URL"; // e.g. https://872ce257.ngrok.io/callback";
```

Comment:
callback url and body vary on each callback, they surely aren't constant, but instead should be as arguments